docs/developers/cookbook.rst

```diff
@@ -0,0 +1,239 @@
+Cheat Sheet
```

Comment:
This cookbook uses DataObsClient (which is the old DO). Are we going to deprecate this in favor of the new DO integration for the 1.0 release? @alasarr

Reply:
Yes, actually I moved it to the 'developers' folder. The idea is to remove old guides (which includes this one), but I didn't know if I should do it in this task or in a different one